themes/godotengine/assets/css/main.css

```diff
@@ -473,7 +473,7 @@ a.card {
   text-decoration: none;
   display: flex;
 }
-a.card:hover {
+.steam-download:hover {
```

Comment:
What about the itch.io download?

Reply:
The itch.io link uses the same class:

https://github.com/godotengine/godot-website/blob/58088e0b505143065ff848d268d1406b4b108752/themes/godotengine/layouts/download.htm#L166